serialization/python_pickle.ksy

```diff
@@ -0,0 +1,588 @@
+    0x53: # "S"
+      id: "string"
+      -orig-id: STRING
+      doc: push string; NL-terminated string argument
+    0x54: # "T"
+      id: "binstring"
+      -orig-id: BINSTRING
+      doc: push string; counted binary string argument
+    0x55: # "U"
+      id: "short_binstring"
+      -orig-id: SHORT_BINSTRING
+      doc: push string; counted binary string argument 256 bytes
+    0x56: # "V"
+      id: "unicode"
+      -orig-id: UNICODE
+      doc: push Unicode string; raw-unicode-escaped argument
+    0x58: # "X"
+      id: "binunicode"
+      -orig-id: BINUNICODE
+      doc: push Unicode string; counted UTF-8 string argument
+    0x61: # "a"
+      id: "append"
+      -orig-id: APPEND
+      doc: append stack top to list below it
+    0x62: # "b"
+      id: "build"
+      -orig-id: BUILD
+      doc: call __setstate__ or __dict__.update()
+    0x63: # "c"
+      id: "global"
+      -orig-id: GLOBAL
+      doc: push self.find_class(modname, name); 2 string args
+    0x64: # "d"
+      id: "dict"
+      -orig-id: DICT
+      doc: build a dict from stack items
+    0x7d: # "}"
+      id: "empty_dict"
+      -orig-id: EMPTY_DICT
+      doc: push empty dict
+    0x65: # "e"
+      id: "appends"
+      -orig-id: APPENDS
+      doc: extend list on stack by topmost stack slice
+    0x67: # "g"
+      id: "get"
+      -orig-id: GET
+      doc: push item from memo on stack; index is string arg
+    0x68: # "h"
+      id: "binget"
+      -orig-id: BINGET
+      doc: push item from memo on stack; index is 1-byte arg
+    0x69: # "i"
+      id: "inst"
+      -orig-id: INST
+      doc: build & push class instance
+    0x6a: # "j"
+      id: "long_binget"
+      -orig-id: LONG_BINGET
+      doc: push item from memo on stack; index is 4-byte arg
+    0x6c: # "l"
+      id: "list"
+      -orig-id: LIST
+      doc: build list from topmost stack items
+    0x5d: # "]"
+      id: "empty_list"
+      -orig-id: EMPTY_LIST
+      doc: push empty list
+    0x6f: # "o"
+      id: "obj"
+      -orig-id: OBJ
+      doc: build & push class instance
+    0x70: # "p"
+      id: "put"
+      -orig-id: PUT
+      doc: store stack top in memo; index is string arg
+    0x71: # "q"
+      id: "binput"
+      -orig-id: BINPUT
+      doc: store stack top in memo; index is 1-byte arg
+    0x72: # "r"
+      id: "long_binput"
+      -orig-id: LONG_BINPUT
+      doc: store stack top in memo; index is 4-byte arg
+    0x73: # "s"
+      id: "setitem"
+      -orig-id: SETITEM
+      doc: add key+value pair to dict
+    0x74: # "t"
+      id: "tuple"
+      -orig-id: TUPLE
+      doc: build tuple from topmost stack items
+    0x29: # ")"
+      id: "empty_tuple"
+      -orig-id: EMPTY_TUPLE
+      doc: push empty tuple
+    0x75: # "u"
+      id: "setitems"
+      -orig-id: SETITEMS
+      doc: modify dict by adding topmost key+value pairs
+    0x47: # "G"
+      id: "binfloat"
+      -orig-id: BINFLOAT
+      doc: push float; arg is 8-byte float encoding
+
+    #'I01\n':
```

Comment:
?

Reply:
This came from an implementation detail in pickle.py. Before `True` and `False` got their own opcodes (`NEWTRUE` and `NEWFALSE`), booleans were encoded by a special value in an `INT` op. I'll remove these, and a note to this effect in the `doc:`.